src/main/Utils/OpenAI.ts

```diff
@@ -0,0 +1,69 @@
+import OpenAi from "openai";
+import { OpenAIRequest } from "../models/OpenAIRequest";
+
+const openai = new OpenAi({
+    apiKey: process.env.OPENAI_API_KEY
+});
+
+//Need to wait until the add job role is complete
+const exampleJson = {
+    location: "Location",
+    capability: "capability",
+    band: "band"
+}
+
+const locationExamples = 'Belfast, Birmingham, Derry, Buenos Aires, Toronto';
+const capabilityExamples = 'Engineering, Platforms, Data and Artificial Intelligence, Cyber Security, Workday, Experience Design, Product, Delivery, Operations, Business Development and Marketing, Organisational Strategy and Planning, People, Commercial and Financial Management, Business Services Support'
+const bandExamples = 'Apperentice, Trainee, Associate, Senior Associate, Consultant, Manager, Principal, Leadership Community'
```

Comment:
would be nice to grab these from the API instead of hardcoding but fine for POC

Reply:
Yeah the get endpoints to get these are part of the add job so I can update it once we get that ticket done